hasher-matcher-actioner/hmalib/lambdas/api/datasets_api.py

```diff
@@ -58,7 +58,7 @@ def to_json(self) -> t.Dict:
 
 
 @dataclass
-class SyncDatasetResponse(JSONifiable):
+class SyncDeleteDatasetResponse(JSONifiable):
```

Comment:
nit: this seems like it is for an API that syncs-deletes  which is probably not what you were going for? Perhaps some other name would do better?

Reply:
yeah I agree! I will change this before I merge it to master